services/libs/data-access-layer/src/old/apps/cache_worker/types.ts

```diff
@@ -4,8 +4,11 @@ export interface ITenant {
 }
 
 export interface ISegment {
+  id: string
   segmentId: string
   slug: string
+  name: string
+  parentName: string
```

Comment:
And these changes are not really needed if you use the right segments query function